pgt/src/PGT/Output/Test/examples/failure-on-invalid-records-count.expected.parsed

```diff
@@ -1 +1 @@
-expected (1 row) comment but received (2 rows)
+expected (1 row) comment but received (2 rows) at line: 6
```

Comment:
Comment: When this exception is thrown and you haven't had any output yet, the line position may not be so useful. I am thinking in a future followup, we should probably write up to error in the golden file while still throwing an error.